src/Diagnostics/atmos_default.jl

```diff
@@ -194,65 +242,87 @@ function compute_diagnosticsums!(
     horzavgs,
     dsums,
 )
-    zvals = CollectedDiagnostics.zvals
     th = thermo_vars(thermoQ[ijk, e])
     ha = horzavg_vars(horzavgs[Nqk * (ev - 1) + k])
     ds = diagnostic_vars(dsums[Nqk * (ev - 1) + k])
 
     u = state.ρu[1] / state.ρ
+    u′ = ha.ρu / ha.ρ
```

Comment:
The prime symbol is confusing to me because it is commonly used for deviation from the mean in fluid dynamics. @akshaysridhar what do you think?

Reply:
How about I rename all `x′` to `avg_x`?